libsolidity/formal/CHC.cpp

```diff
@@ -525,6 +527,39 @@ void CHC::visitAssert(FunctionCall const& _funCall)
 	m_context.addAssertion(m_error.currentValue() == previousError);
 }
 
+void CHC::internalFunctionCall(FunctionCall const& _funCall)
+{
+	solAssert(m_currentContract, "");
+
+	auto const* function = functionCallToDefinition(_funCall);
+	if (function)
+	{
+		if (m_currentFunction && !m_currentFunction->isConstructor())
+			m_callGraph[m_currentFunction].insert(function);
+		else
+			m_callGraph[m_currentContract].insert(function);
+		auto const* contract = function->annotation().contract;
+
+		// Liraries can have constants as their "state" variables,
```

Comment:
To me it keeps the code changes to a minimum level. Otherwise we'd need to add code that analyzes that separately, create new predicates/variables, etc.

Reply:
Yeah, in any case, it's fine for now.